README.md

```diff
@@ -223,38 +223,40 @@ The following is complete set of connector sink properties-
-| 23 | errors.deadletterqueue.sasl.mechanism | Channel to write records that failed deserialization | SASL mechanism of secure Kafka cluster<br>*Optional but when feature is used with secure cluster, is required*  |
-| 24 | errors.deadletterqueue.sasl.jaas.config | Channel to write records that failed deserialization | JAAS config of secure Kafka cluster<br>*Optional but when feature is used with secure cluster, is required*  |
-| 25 | misc.deadletterqueue.bootstrap.servers | Channel to write records that due to reasons other than deserialization | CSV of kafkaBroker:port <br>*Optional*  |
-| 26 | misc.deadletterqueue.topic.name | Channel to write records that due to reasons other than deserialization | Pre-created topic name <br>*Optional*  |
-| 27 | misc.deadletterqueue.security.protocol | Channel to write records that due to reasons other than deserialization  | Securitry protocol of secure Kafka cluster <br>*Optional but when feature is used with secure cluster, is required*  |
-| 28 | misc.deadletterqueue.sasl.mechanism | Channel to write records that due to reasons other than deserialization | SASL mechanism of secure Kafka cluster<br>*Optional but when feature is used with secure cluster, is required*  |
-| 29 | misc.deadletterqueue.sasl.jaas.config | Channel to write records that due to reasons other than deserialization | JAAS config of secure Kafka cluster<br>*Optional but when feature is used with secure cluster, is required*  |
-| 30 | consumer.override.bootstrap.servers | Security details explicitly required for secure Kafka clusters  | Bootstrap server:port CSV of secure Kafka cluster <br>*Required for secure Kafka clusters*  |
-| 31 | consumer.override.security.protocol | Security details explicitly required for secure Kafka clusters  | Security protocol of secure Kafka cluster <br>*Required for secure Kafka clusters*  |
-| 32 | consumer.override.sasl.mechanism | Security details explicitly required for secure Kafka clusters | SASL mechanism of secure Kafka cluster<br>*Required for secure Kafka clusters*  |
-| 33 | consumer.override.sasl.jaas.config | Security details explicitly required for secure Kafka clusters | JAAS config of secure Kafka cluster<br>*Required for secure Kafka clusters*  |
-| 34 | consumer.override.sasl.kerberos.service.name | Security details explicitly required for secure Kafka clusters, specifically kerberized Kafka | Kerberos service name of kerberized Kafka cluster<br>*Required for kerberized Kafka clusters*  |
-| 35 | consumer.override.auto.offset.reset | Configurable consuming from offset | Possible values are - earliest or latest<br>*Optional*  |
-| 36 | consumer.override.max.poll.interval.ms| Config to prevent duplication | Set to a value to avoid consumer leaving the group while the Connector is retrying <br>*Optional*  |
+| 5 | aad.auth.authority | Credentials for Kusto | Provide the tenant ID of your Azure Active Directory<br>*Required if managed identity is disabled*  |
```

Comment:
@abestel ,

 Roll up the version in pom.xml

Reply:
Done